### ISSUE TITLE
Dropdowns in course/dept detail component missing label

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs-admin#195]
The department dropdown form element does not have a`label` attribute.
See #192 for why this is an accessibility issue.

**Comment by kburk1997** [comment]
This is also reflected in the school dropdown in the departments form.
